src/Servers/HttpSys/src/RequestProcessing/RequestContext.FeatureCollection.cs

```diff
@@ -742,7 +742,7 @@ void IConnectionLifetimeNotificationFeature.RequestClose()
             // Set the connection close feature if the response hasn't sent headers as yet
             if (!Response.HasStarted)
             {
-                Response.Headers["Connection"] = "close";
+                Response.Headers[HeaderNames.Connection] = "close";
```

Comment:
```suggestion
                Response.Headers.Connection = "close";
```

Reply:
I tried this at first, but got error. If I'm not wrong the reason is `Connection` is a default interface property and is implicit and here `Response.Headers` type is `HeaderCollection` and not `IHeaderDictionary`.

Reply:
Ah